tock/tock/templates/hours/timecard_form.html

```diff
@@ -56,14 +76,20 @@
     {% endfor %}
   </div>
   {% if formset.errors %}
-    <div class="alert-red"><b>{{ formset.non_form_errors }}</b></div>
+    <div class="usa-alert usa-alert-error">
+      <div class="usa-alert-body">
+        <p class="usa-alert-heading"><strong>{{ formset.non_form_errors }}</strong></p>
+      </div>
+    </div>
   {% endif %}
   {% if messages %}
-  <ul class="messages">
     {% for message in messages %}
-      <b><li{% if message.tags %} class="alert-{{ message.tags }}"{% endif %}>{{ message }}</li></b>
+    <div class="usa-alert usa-alert-info">
```

Comment:
Yeah, this was a shortcut b/c I know the tags aren't being use. Should definitely get this updated later on.